### ISSUE TITLE
[BUG]: Cant search in Arabic, cant change the keyboard language.

### ISSUE THREAD

**Issue by mohamadat** [yuliskov/SmartTube#3852]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/yuliskov/SmartTube/issues) or [closed](https://github.com/yuliskov/SmartTube/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://github.com/yuliskov/SmartTube#faq) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.

### Affected version

latest Beta

### Device Type

Smart TV/Box

### Affected Android

Android TV

### Steps to reproduce the bug

Go to search
Click on the search Icon

**Comment by Khalid-Shammout** [comment]
yes, that's because SmartTube relies on the system installed keyboard instead of it's own, so technicaly if there is a keyboard that works on smart tv's with multiple possible languages that should be done. However it is a good idea to have a built in keyboard
